### PR TITLE
Change log for latest release

### PR DESCRIPTION
**Pull request by Athanaseus**
Preparing 0.1.4 release
* Includes package documentation

### CODE REVIEW

**Comment by ajoubertza** [comment]
Great, just check the docs for getting started are accurate.  E.g. for installing from PYPI is no longer "in the near future", and remove duplication from the readme.

File: tango_simlib/examples/weather3.py [hunk 1]
```diff
@@ -4,11 +4,10 @@
 
 
 def main():
-    sim_data_files = ['../tests/weather_sim.xmi', '../tests/weather_SIMDD_3.json']
+    sim_data_files = ['../tests/weather.xmi', '../tests/Weather_SIMDD_3.json']
```
Comment (ajoubertza) [inline tango_simlib/examples/weather3.py]: I think you need a capital "W" for the first one as well.  The file is `Weather.xmi`.

File: doc/intro.rst [hunk 2]
```diff
@@ -138,15 +141,16 @@ Weather simulators
...
 with a ``SimControl`` instance using the ``tango_launcher.py`` script.
+Change directory to tango_simlib/examples/
 
 .. code-block:: bash
 
     $ tango-simlib-tango-launcher --name mkat_sim/weather/1 --class Weather\
                           --name mkat_simcontrol/weather/1\
                           --class WeatherSimControl\
-                          --server-command tango-simlib-weather-xmi-DS --port 0\
+                          --server-command Weather1.py --port 0\
```
Comment (ajoubertza) [inline doc/intro.rst]: The actual Python files in the examples folder don't have a capital "W".  For example, `weather1.py`.  Same for the next example.

Comment (Athanaseus) [inline doc/intro.rst]: Duly

**Comment by Athanaseus** [comment]
@ajoubertza / @kmadisa , Ready for release.

**Pull request merged**